### DISCUSSION TITLE
Feature Request: User Override

### DISCUSSION THREAD

**Discussion by TFWIsaac** [JakyeRU/Larascord#97] [Ideas]
Hey,

as in the title mentioned, i would like to have a user override. That means that in the config.php a new key 'user_override' exists, where you can add in an array discord user_ids.

For example my disord user_id is in this array, then the check (if enabled) will be skipped. Any user who is in that array can login without any further checks or so.

**Comment by ValentinNeaga** [comment]
Do you mean a new setting called something like "user_override" that will allow users to log into the application even if they don't meet the necessary requirements, for example, having a role in a guild, or being a member of a guild?

Or something like a whitelist, where only certain users can log in at all?

**Reply by TFWIsaac** [reply]
The first one you have mentioned :)

**Reply by ValentinNeaga** [reply]
Alright. I'll consider your idea when creating future updates. Thank you.
